react-app/src/components/ProposalDetailScreen/ProposalDetailScreen.tsx

```diff
@@ -65,6 +65,7 @@ const ProposalDetailScreen: React.FC = () => {
       }
 
       try {
+        toast.info(translate("ProposalDetail.setReaction.requesting"));
```

Comment:
That means every time we set reaction it would show this, I think we only want to show this when we actually request for sign in? i.e `authErrorLink` in AppApolloProvider?